graspy/version/version.txt

```diff
@@ -0,0 +1 @@
+0.1.0.dev20200924175647
```

Comment:
@alyakin314 this is a new thing, this file gets updated to specify that you are running a local copy when developing. we may ask you to remove this change... but @dwaynepryce do you have any thoughts? This seems like the kind of thing that people will accidentally commit often, so wondering how we should handle in general, or if it is even a problem?

Reply:
can we .gitignore it?

Reply:
we do want it tracked generally speaking since we'll have to update this file I think

Reply:
okay